frontend/packages/container-security/src/kebab-actions.ts

```diff
@@ -0,0 +1,36 @@
+import { K8sKind, PodKind } from '@console/internal/module/k8s';
+import { referenceForModel } from '@console/internal/module/k8s/k8s';
+import { KebabAction, KebabOption } from '@console/internal/components/utils/kebab';
+import { PodModel } from '@console/internal/models';
+import * as _ from 'lodash';
+import { ImageManifestVulnModel } from './models';
+
+const listPathFor = (namespace: string, imageID: string) =>
+  [
+    '/k8s',
+    namespace === '' ? 'all-namespaces' : `ns/${namespace}`,
+    referenceForModel(ImageManifestVulnModel),
+    `?name=sha256.${imageID.split('sha256:')[1]}`,
+  ].join('/');
+
+const ViewImageVulnerabilities = (model: K8sKind, obj: PodKind): KebabOption => {
+  const ready = _.get(obj.status, 'containerStatuses', []).length > 0;
```

Comment:
```suggestion
  const ready = (obj.status?.containerStatuses || []).length > 0;
```

Reply:
I forgot that we can use optional chaining now!

Reply:
@alecmerdler You'll need to rebase the PR for the eslint / prettier updates to use optional chaining, #3747